x-pack/plugins/cross_cluster_replication/public/app/services/follower_index_default_settings.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const defaultSettings = {
+  maxReadRequestOperationCount: 5120,
+  maxOutstandingReadRequests: 12,
+  maxReadRequestSize: '32mb',
+  maxWriteRequestOperationCount: 5120,
+  maxWriteRequestSize: '9223372036854775807b',
+  maxOutstandingWriteRequests: 9,
+  maxWriteBufferCount: 2147483647,
+  maxWriteBufferSize: '512mb',
+  maxRetryDelay: '500ms',
+  readPollTimeout: '1m',
+};
+
+export const getSettingDefault = (name) => {
+  if(!defaultSettings[name]) {
+    throw new Error(`Unknown setting ${name}`);
```

Comment:
I think this label should be translated as well.

Reply:
@jen-huang and I discussed this (https://github.com/elastic/kibana/pull/29153#discussion_r250707919) and agreed not to internationalize it, but we can change our minds. 😄  Does the i18n team plan to localize all of these types of errors throughout Kibana? If so, could you explain the rationale?